nms/app/packages/magmalte/app/views/traffic/ApnOverview.js

```diff
@@ -146,7 +145,17 @@ export default function ApnOverview() {
               {title: 'Qos Profile', field: 'qosProfile', type: 'numeric'},
               {title: 'Added', field: 'added', type: 'datetime'},
             ]}
-            menuItems={[{name: 'Edit'}, {name: 'Deactivate'}, {name: 'Remove'}]}
+            handleCurrRow={(row: ApnRowType) => setCurrRow(row)}
+            menuItems={[
+              {
+                name: 'Edit JSON',
```

Comment:
minor nit: might be cleaner to just call this "Edit" instead of "Edit JSON". We can do this cleanup as a part of another PR. I will merge this for now.